src/constants/index.ts

```diff
@@ -45,6 +45,13 @@ export const FRAX_3_POOL_NAME = "Frax 3Pool"
 export const TBTC_METAPOOL_V2_BTCV2_V3_NAME = "tBTCv2-BTCv2_v3"
 export const WCUSD_METAPOOL_V2_USDV2_V3_NAME = "wcUSD-USDv2_v3"
 export const SUSD_METAPOOL_V2_USDV2_V3_NAME = "sUSD-USDv2_v3"
+export const FRAX_USDC_POOL_NAME = "Saddle FRAX Pool"
+export const FRAX_USDT_METAPOOL_NAME = "frax-USDT Meta"
+export const FRAX_SUSD_METAPOOL_NAME = "frax-sUSD Meta"
+export const FRAX_ALUSD_METAPOOL_NAME = "frax-alUSD Meta"
+export const ARB_FRAX_USDS_METAPOOL_NAME = "frax-USDs Meta"
+export const FTM_FRAX_USDT_METAPOOL_NAME = "frax-USDT Meta"
+export const FTM_FRAX_ALUSD_METAPOOL_NAME = "frax-alUSD Meta"
```

Comment:
FTM pools seperate due to pool token composition order being diff from Saddle FRAX base pool. Added as new entries to pool array.